composer.json

```diff
@@ -12,20 +11,24 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0"
+        "php": "^5.5 || ^7.0"
```

Comment:
Hm, that is interesting. I've never thought about that. 
You are correct but I believe we should make this change anyways. I consider it a bug that we say we support PHP8 (because we do not) and bugs should be fixed.